src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/TypesTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Sql.Types;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.IpcTests
+{
+    [Collection("Spark E2E Tests")]
+    public class TypesTests
+    {
+        private static IJvmBridge Jvm { get; } = SparkEnvironment.JvmBridge;
+
+        [Theory]
+        [InlineData("null")]
+        [InlineData("string")]
+        [InlineData("binary")]
+        [InlineData("boolean")]
+        [InlineData("date")]
+        [InlineData("timestamp")]
+        [InlineData("double")]
+        [InlineData("float")]
+        [InlineData("byte")]
+        [InlineData("integer")]
+        [InlineData("long")]
+        [InlineData("short")]
+        public void TestSimpleTypes(string typeName)
+        {
+            JvmObjectReference atomicType = DataType.FromJson(Jvm, $@"""{typeName}""");
```

Comment:
instead of passing a string, can you construct the DataType and call the .Json method instead ?

Reply:
yes, let's do that.

Reply:
I have changed to `ClassData`, so that we can pass `SimpleTypes` as `DataType`.